lib/types/index.ts

```diff
@@ -1,5 +1,19 @@
-export interface Config {
-  msg: string;
-  count: number;
-  interval?: number;
+import { ALL_CHANNELS, ALL_DB_ENGINES } from '../consts';
+
+export type DBEngine = (typeof ALL_DB_ENGINES)[number];
```

Comment:
trivial comment/can be skipped: I always try to avoid fancy declaration or one liners, so that after 6 months if i look back at the code i don't need to google the syntax / spend time to understand it